modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts

```diff
@@ -225,3 +225,72 @@ test('LPD-31669 Check whether admin user is redirected to organization page afte
 		await organizationUsersPage.usersTableRowLink(userName)
 	).toBeVisible();
 });
+
+test('LPD-31645 Search by Organizations when setting a users organization roles', async ({
+	apiHelpers,
+	editUserPage,
+	page,
+	usersAndOrganizationsPage,
+}) => {
+	const user = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	const organization1 = await apiHelpers.headlessAdminUser.postOrganization();
+	const organization2 = await apiHelpers.headlessAdminUser.postOrganization();
+
+	await apiHelpers.headlessAdminUser.assignUserToOrganizationByEmailAddress(
+		organization1.id,
+		user.emailAddress
+	);
+
+	apiHelpers.data.push({
+		id: `${organization1.id}_${user.emailAddress}`,
+		type: 'organizationUserAccountAssociation',
+	});
+
+	await apiHelpers.headlessAdminUser.assignUserToOrganizationByEmailAddress(
+		organization2.id,
+		user.emailAddress
+	);
+
+	apiHelpers.data.push({
+		id: `${organization2.id}_${user.emailAddress}`,
+		type: 'organizationUserAccountAssociation',
+	});
+
+	await usersAndOrganizationsPage.goToUsers();
+	await (
+		await usersAndOrganizationsPage.usersTableRowLink(user.alternateName)
+	).click();
+
+	await editUserPage.rolesLink.click();
+	await editUserPage.selectOrganizationRolesButton.click();
+
+	await page.waitForTimeout(500);
+
+	expect(
+		(await editUserPage.selectOrganizationRolesTable.getByRole('row').all())
+			.length
+	).toEqual(3);
+
+	await editUserPage.selectOrganizationRolesSearchBar.fill(
+		organization1.name
+	);
+	await editUserPage.selectOrganizationRolesSearchBarButton.click();
+
+	await page.waitForTimeout(500);
```

Comment:
why this? probably the same as before but there is any other element we can use to avoid this "fixed" timeout?